PutJSONTogether/app.js

```diff
@@ -0,0 +1,70 @@
+var fs = require("fs");
+console.log("\n *START* \n");
+var content = "artists.json";
+
+var musicians = [];
+var newObj = {};
+
+function convertToInformation(content) {
+    fs.readFile(content, 'utf8', function readFileCallback(err, data){
+        if (err){
+            console.log(err);
+        } else {
+            obj = JSON.parse(data); //now it an object
+            arrayofObj = obj.table.length;
+
+            for(i = 0; i < obj.table.length; i++) {
+                newObj = {};
+                if (obj.table[i].hasOwnProperty("aboutartist")) {
+                    newObj.name = obj.table[i].aboutartist.name;
+                    newObj.nationality = null;
+
+                    if (obj.table[i].aboutartist.hasOwnProperty("life-span")) {
+                        if(obj.table[i].aboutartist["life-span"].hasOwnProperty("begin")) {
+                            if (obj.table[i].aboutartist["life-span"].begin != null) {
+                                newObj.dateOfBirth = obj.table[i].aboutartist["life-span"].begin;
+                            } else { newObj.dateOfBirth = null; } 
+                        } else { newObj.dateOfBirth = null; }
+                    } else { newObj.dateOfBirth = null; }
```

Comment:
Why don't you set the DoB null before the if, instead of having three else blocks?